lib/Importer/CMakeLists.txt

```diff
@@ -9,8 +9,9 @@ if(NOT TARGET onnx_proto)
     set(ONNX_NAMESPACE "glow_onnx")
   endif()
   add_definitions("-DONNX_NAMESPACE=${ONNX_NAMESPACE}")
-  add_subdirectory(${GLOW_THIRDPARTY_DIR}/onnx EXCLUDE_FROM_ALL build)
+  add_subdirectory(${GLOW_THIRDPARTY_DIR}/onnx EXCLUDE_FROM_ALL build_onnx)
 endif()
+add_subdirectory(${GLOW_THIRDPARTY_DIR}/foxi build_foxi)
```

Comment:
This isn't needed for onnx proto, may be better to `target_include_directories` on Importer? not sure